components/base/Collapsible.tsx

```diff
@@ -0,0 +1,85 @@
+/** @jsx jsx */
+import ReactCollapsible from "react-collapsible";
+import styled from "@emotion/styled";
+import { css, jsx } from "@emotion/core";
+import plusIcon from "../../public/images/plus-circle-icon.svg";
+import minusIcon from "../../public/images/minus-circle-icon.svg";
+import { GREY_2, WHITE_PRIMARY } from "../colors";
+import { POPPINS } from "../typography";
+import { BOX_SHADOW_7 } from "../shadowDepths";
+
+// Styles
+const CollapsibleWrapper = styled.div`
+  /* Main container element for react-collapsible https://www.npmjs.com/package/react-collapsible#collapsible */
```

Comment:
Overriding **react-collapsible**'s built in classes `.Collapsible` and `.Collapsible__contentInner`